.github/workflows/ci.yaml

```diff
@@ -1653,11 +1653,12 @@ jobs:
                 "type": "section",
                 "text": {
                   "type": "mrkdwn",
-                  "text": "<@U08TJ4YNCA3> investigate this CI failure. Check logs, search for existing issues, use git blame to find who last modified failing tests, create issue in coder/internal (not public repo), use title format \"flake: TestName\" for flaky tests, and assign to the person from git blame."
+                  "text": "<@U08TJ4YNCA3> '"${BLINK_CI_FAILURE_PROMPT}"'"
```

Comment:
This will break when someone puts a double quote in the prompt. You'll need to use something more reliable like `jq` to set the value or simply just escape it